stock_valuation_fifo_lot/wizard/stock_valuation_layer_revaluation.py

```diff
@@ -0,0 +1,244 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import Command, _, api, fields, models
+from odoo.exceptions import UserError
+from odoo.tools import float_compare, float_is_zero
+
+
+class StockValuationLayerRevaluation(models.TransientModel):
+    _inherit = "stock.valuation.layer.revaluation"
+
+    current_value_lot_svl = fields.Float(
+        "Current Value ", compute="_compute_current_value_lot_svl"
+    )
+    current_quantity_lot_svl = fields.Float(
+        "Current Quantity", compute="_compute_current_value_lot_svl"
+    )
+    lot_id = fields.Many2one(
+        "stock.lot",
+        "Lot/Serial Number",
+    )
+    display_lot_id = fields.Boolean(compute="_compute_display_lot_id")
+
+    def _search_svl_move_lines(self, lot_id):
```

Comment:
Wouldn't it be more efficient to directly search `stock.move.line` instead of getting records via SVL?